docs/_docs/2. functional.md

```diff
@@ -33,12 +33,9 @@ split into those covering _required_ vs _optional_ features.
 
 ### Summary of results
 
-The chart below uses the **score** from the **overall** column in the [Summary results table](#summary-of-results) below to visually 
-present the feature completeness of each validator implementation.
+The charts below show the compliance with **required** and **optional** JSON Schema specification features of each validator implementation.
```

Comment:
```suggestion
The charts below illustrate the percentage of required and optional test classes that pass for each draft of the JSON Schema, and aggregate/overall graphs that show the pass rate across all the drafts an implementation supports.  Results are broken down into separate _required_ and _optional_ graphs. 
```

Maybe?

Reply:
Assuming you meant "required and optional test **cases**"